packages/react-table/src/components/Table/examples/LegacyTableActions.tsx

```diff
@@ -64,10 +135,12 @@ export const LegacyTableActions: React.FunctionComponent = () => {
     'Branches',
     'Pull requests',
     'Workspaces',
-    'Last commit'
+    'Last commit',
+    { title: '', dataLabel: 'testing', cellTransforms: [fitContent] }
   ];
+
   const rows: TableProps['rows'] = repositories.map(repo => {
-    const cells = [repo.name, repo.branches, repo.prs, repo.workspaces, repo.lastCommit];
+    const cells = [repo.name, repo.branches, repo.prs, repo.workspaces, repo.lastCommit, repo.empty];
```

Comment:
Maybe instead we can just put the whole `{ title, cellTransforms }` object here in the cells array instead of `repo.empty`, since it is the same for all rows anyway?